taichi/backends/device.h

```diff
@@ -385,6 +385,7 @@ struct PipelineSourceDesc {
 
 // FIXME: this probably isn't backend-neutral enough
 enum class AllocUsage : int {
+  Transfer = 0,
```

Comment:
Actually, let's append to this list and use `16`? `0` should be reserved if we are using it as a bitmask.

Reply:
This is intended, with usage of none we default it to transfer

Reply:
Maybe actually should just call it none? As we assume transfer is always possible on all buffers.